<launch>
  <arg name="rate" default="30.0"/>
  <arg name="robot_namespace" default="xmaxx1"/>

  <node pkg="controller_adaptiveclbf" type="figure_eight.py" name="figure_eight" output="screen" respawn="true" ns="$(arg robot_namespace)">
    <remap from="target" to="odometry_goal" />
  </node>
</launch>
